bitbots_bringup/README

```diff
@@ -1,377 +0,0 @@
-pylon 5 Camera Software Suite for Linux for Use with Basler
```

Comment:
Do we have this documentation in a more sensible place?

Reply:
basler has it